doc/developer-notes.md

```diff
@@ -249,6 +250,35 @@ Development tips and tricks
 Run configure with `--enable-debug` to add additional compiler flags that
 produce better debugging builds.
 
+### Show sources in debugging
+
+If you have ccache enabled absolute paths are stripped from debug information
+with -fdebug-prefix-map and -fmacro-prefix-map options (if supported by the 
```

Comment:
```suggestion
with -fdebug-prefix-map and -fmacro-prefix-map options (if supported by the
```

from the linter

Reply:
Thanks for extracting this out from the logs. Fixed